internal/daemon/daemon_test.go

```diff
@@ -7,20 +7,31 @@ import (
 	"time"
 
 	"github.com/lunarway/strong-duckling/internal/daemon"
-	"github.com/lunarway/strong-duckling/internal/test"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDaemon_Loop(t *testing.T) {
-	logger := test.NewLogger(t)
 	tickInterval := 10 * time.Millisecond
 	testDuration := 100 * time.Millisecond
 	expectedTickCount := int32(10)
 
 	var actualTickCount int32
 	d := daemon.New(daemon.Configuration{
 		Interval: tickInterval,
-		Logger:   logger,
+		Probes: &daemon.Probes{
+			Started: func(time.Duration) {
+				t.Log("Daemon started")
```

Comment:
Would be nice with the `started`, `stopped`, etc. as a field in the log.

Reply:
In a test?

Reply:
I've added a `state` field to the main logs instead with values `started` and `stopped`.